.github/workflows/libevm-delta.yml

```diff
@@ -9,35 +9,36 @@ on:
 
 jobs:
   diffs:
-    env:
-      # Last commit from rename-module workflow job to be included in `main`
-      LIBEVM_BASE: 0b56af5a01b8a0c6fc9d60247bb79ffd03d1bcfd
```

Comment:
(informational) Confirmed that this commit hash is still [used by the workflow](https://github.com/ava-labs/libevm/actions/runs/12751809929/job/35539746488?pr=99#step:5:3).